modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -130,7 +130,7 @@ export class Configuration {
     }
 
     get fetchApi(): FetchAPI {
-        return this.configuration.fetchApi || window.fetch.bind(window);
+        return this.configuration.fetchApi || (typeof window !== 'undefined' ? window.fetch.bind(window) : fetch );
```

Comment:
the symbol `fetch` isn't declared or defined anywhere in this file, so this should be a compilation error, right?

Reply:
No, `fetch` is available on global scope, if you have a shim in nodejs:
```
global.fetch = global.fetch || require('node-fetch')
```
Similarly to `FormData`:
```
global.FormData = global.FormData || require('form-data')
```

Reply:
but what if `fetch` is not defined in the global scope? or is it always available in the global scope? if so, why use `window.fetch` at all?

Reply:
according to https://developer.mozilla.org/de/docs/Web/API/WindowOrWorkerGlobalScope/fetch `fetch` should be available in any context

Reply:
If it is really available in the global scope in every browser (checked in Firefox & Chromium) & in node, I'd be in favor of only checking fetch if `configuration.fetchApi' isn't defined, i.e.

```
return this.configuration.fetchApi || fetch; 
```

Reply:
@hello-brsd can you implement the change suggested by @TiFu?